packages/web3-core-method/src/index.js

```diff
@@ -360,14 +360,14 @@ Method.prototype._confirmTransaction = function (defer, result, payload) {
 
                     } else {
                         if(receipt) {
-                            receipt = JSON.stringify(receipt, null, 2);
+                            var receiptString = JSON.stringify(receipt, null, 2);
```

Comment:
When I run the tests i got this JSHint error at the end: 
```
packages/web3-core-method/src/index.js: line 366, col 104, 'receiptString' used out of scope.
packages/web3-core-method/src/index.js: line 370, col 102, 'receiptString' used out of scope.
```
Could you please move the ```var``` declaration out of the ```if``` block?